### PR TITLE
fix: update bookedcount

### PR DESCRIPTION
**Pull request by tanush077**
Sorry for missing it ! 
#8

### CODE REVIEW

File: routes/bookings/CreateBooking.go [hunk 1]
```diff
@@ -32,7 +32,27 @@ func CreateBooking(c *fiber.Ctx) error {
...
+	}
+
+	// Check if there are available seats
+	if ride.BookedSeats >= ride.TotalSeats {
+		log.Println("No available seats for this ride")
+		return c.Status(400).SendString("No available seats for this ride")
+	}
+
+	// Update the booked seats count
+	ride.BookedSeats++
```
Comment (RohanKhatua) [inline routes/bookings/CreateBooking.go:48]: This should be done in the update db operation.

File: routes/bookings/CreateBooking.go [hunk 2]
```diff
@@ -32,7 +32,27 @@ func CreateBooking(c *fiber.Ctx) error {
 		log.Println("Similar booking already exists")
 		return c.Status(400).SendString("Similar booking already exists")
 	}
+	ride := models.Ride{}
+	if err := database.Database.Db.Where("id = ?", Booking.RideID).First(&ride).Error; err != nil {
```
Comment (RohanKhatua) [inline routes/bookings/CreateBooking.go:36]: If you are only checking one condition it does not need a where, just use first or find with conditions

**Review by RohanKhatua — CHANGES_REQUESTED** [review]
Please see review comments

**Comment by tanush077** [comment]
I have made the required changes #12

**Review by RohanKhatua — APPROVED** [review]
LGTM

**Pull request merged**